b1ddi/resource_dns_record_test.go

```diff
@@ -105,14 +105,15 @@ func TestAccResourceDnsRecord_full_config(t *testing.T) {
 						primary_type = "cloud"
 						view = b1ddi_dns_view.tf_acc_test_dns_view.id
 					}
+					resource "b1ddi_dns_auth_zone" "tf_acc_test_dns_record_reverse_zone" {
+						fqdn         = "1.168.192.in-addr.arpa."
+  						primary_type = "cloud"
+  						view         = b1ddi_dns_view.tf_acc_test_dns_view.id	
+					}
 					resource "b1ddi_dns_record" "tf_acc_test_dns_record" {
 						comment = "This DNS Record is created by the terraform provider acceptance test"
 						disabled = true
 						name_in_zone = "tf_acc_test_a_record"
-						options = {
-							"create_ptr" = true
- 							"check_rmz" = true
-						}
```

Comment:
Did we have tests for those values somewhere else?

Reply:
The problem is options aren't part of the DNS record read API response. So we cannot test this

Reply:
I don't know the framework well enough to see if we can still them those values. I hope they work in the real world and by removing them we don't remove something which would detect regressions in the future.

Reply:
This isn't an issue with the framework per se. DNS record create API allows us to set the options and the read API doesn't return the set options in the response(I don't know why we do not return the options). In the current state, the acceptance will fail. So updated it to remove fields that don't show in the response.